static/app/views/replays/list.tsx

```diff
@@ -55,6 +50,19 @@ function ReplaysListContainer() {
           <Layout.Main fullWidth>
             <LayoutGap>
               <ReplayListPageHeaderHook />
+              {allMobileProj && mobileBetaOrg ? (
+                <StyledAlert icon={<IconInfo />} showIcon>
+                  {tct(
+                    `[strong:Mobile Replay is now generally available.] Orgs that participated in the beta will have a two month grace period of unlimited usage until March 6. After that, you will be billed for [link:additional replays not included in your plan].`,
```

Comment:
We're only showing this to those who participated, right? how about:

```suggestion
                    `[strong:Mobile Replay is now generally available.] Since you participated in the beta, will have a two month grace period of free usage, until March 6. After that, you will be billed for [link:additional replays not included in your plan].`,
```